Main.java

```diff
@@ -3,4 +3,86 @@ public static void main(String[] args){
         System.out.println ("Test");
     }
 
+    //sleapIn
+    public boolean sleepIn(boolean weekday, boolean vacation) {
+        return (!weekday || vacation);
+    }
+
+    //monkey trouble
+    public boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
+        if (aSmile && !bSmile || !aSmile && bSmile) {
+            return false;
+        } return true;
+    }
+
+    //sumDouble
+    public int sumDouble(int a, int b) {
+        if (a==b) {
+            return (a+b)*2;
+        } return (a+b);
+    }
+
+    //diff21
+    public int diff21(int n) {
+        if (n<21) {
+            return 21-n;
+        }return (n-21)*2;
+    }
+
+    //parrotTrouble
+    public boolean parrotTrouble (boolean talking, int hour) {
+        return (talking && (hour <7 || hour >20));
+    }
+
+    //makes10
+    public boolean makes10(int a, int b) {
+        return ((a+b)==10 || a==10 || b==10);
+    }
+
+    //nearHundred
+    public boolean nearHundred (int n) {
+        return (Math.abs(n-100)<=10 || Math.abs(n-200)<=10);
+    }
+
+    //posNeg
+    public boolean posNeg (int a, int b, boolean negative) {
+        return a<0 && b>0 && !negative || a>0 && b<0 && !negative || a<0 && b<0 && negative;
+    }
+
+    //notString
+    public String notString (String str) {
+        if (str.length() >=3 && str.substring (0, 3).equals("not")) {
```

Comment:
possible NPE on str variable

also, it is a good practice to use:
constant.equals(sth) because it is potentially safer, so in your case:
"not".equals(...)

what is more, you can also use ?:

optionally, you can store "not" in a variable as you use it twice